### DISCUSSION TITLE
Unusual laptop/optimus behaviour using Laser

### DISCUSSION THREAD

**Discussion by djenson2000** [ppy/osu#18371] [Q&A] [answered]
On other games where I have selected the dGPU, I see no usage on the iGPU, yet with Laser:
![image](https://user-images.githubusercontent.com/86269857/169866358-f97c91c7-028d-47d6-bc4f-6f0df5e41cb3.png)

Specs: 
i7-10875H 
1660ti
16GB 3200Mhz

Is this normal expected behaviour, or is my laptop messing up optimus somehow

Sorry if this is the wrong place to ask questions like this.

[database.log](https://github.com/ppy/osu/files/8756477/database.log)
[network.log](https://github.com/ppy/osu/files/8756478/network.log)
[runtime.log](https://github.com/ppy/osu/files/8756480/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8756481/updater.log)
[performance.log](https://github.com/ppy/osu/files/8756479/performance.log)

**Comment by frenzibyte** [comment]
Would prefer seeing a screenshot of which task is using iGPU and which is using dGPU here (mainly whether it's `Desktop window manager` that's taking iGPU usage), but this maybe related to https://github.com/ppy/osu/issues/5582 generally.

**Comment by djenson2000** [comment]
Sorry about that screenshot. got some new ones using "always on top"
osu open:
![Osu Open](https://user-images.githubusercontent.com/86269857/169873920-cfb0e152-0dce-492b-9af9-19a67a92426c.png)
On idle:
![Idle Useage](https://user-images.githubusercontent.com/86269857/169873937-ead41d8d-6367-4873-aab4-741277268a32.png)

**Comment by peppy** [comment, marked as answer]
You're pushing too many frames. With other games you're probably doing 60fps and therefore not straining your intel GPU at all.

Read about optimus if you care to understand.

**Reply by djenson2000** [reply]
But why would my intel gpu be strained, i'm using the 1660ti or so i thought? 288FPS is too much? I'm not getting any weird stutters, i just thought it was weird behavior. 

It is just because frames are having to go from dGPU --> iGPU Frame buffer --> Display?

**Reply by peppy** [reply]
yes. set your frame limiter to 60 if you care.
